src/inc/Admin/views/generate.php

```diff
@@ -50,7 +50,7 @@
 			</tbody>
 		</table><br>
 
-		<h2 class="title">
+		<h2 class="seed">
```

Comment:
```suggestion
		<h2 class="title">
```

The CSS needs to be `title`.